src/Microsoft.AspNet.Mvc/MvcServiceCollectionExtensions.cs

```diff
@@ -44,7 +44,6 @@ public static IMvcBuilder AddMvc(this IServiceCollection services, Action<MvcOpt
 
             builder.AddApiExplorer();
             builder.AddAuthorization();
-
             // Order added affects options setup order
```

Comment:
The change makes it less clear what the comment refers to. If it doesn't go away when you next rebase, revert changes to this file.